src/app/dev/DevToys.Tools/Helpers/XmlHelper.cs

```diff
@@ -0,0 +1,129 @@
+﻿using System.Text;
+using System.Text.RegularExpressions;
+using DevToys.Tools.Models;
+using System.Xml;
+using Microsoft.Extensions.Logging;
+using DevToys.Api.Core;
```

Comment:
Please try `Ctrl+R, Ctrl+G` to clean unused `using` and reorder them automatically. :)